data/tests/test_emission_calculations.py

```diff
@@ -22,12 +22,18 @@ class TestEmissionCalculations(unittest.TestCase):
 
     def test_get_n_prep_data_from_smhi(self):
         path_input_df = "tests/reference_dataframes/df_municipalities.xlsx"
-        path_expected_df = "tests/reference_dataframes/df_smhi.xlsx"
 
         df_input = pd.DataFrame(pd.read_excel(path_input_df))
         df_result = get_n_prep_data_from_smhi(df_input)
-        df_expected = pd.DataFrame(pd.read_excel(path_expected_df))
-        pd.testing.assert_frame_equal(df_result, df_expected, check_dtype=False)
+        result_columns = df_result.columns.to_list()[4:] # Skip the first 4 columns which is the 'index'
+        expected_columns = [1990, 2000, 2005, 2010, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022]
+
+        # Check that the expected columns are in the dataframe
+        assert result_columns == expected_columns
+
+        # Each of the column values should all be greater than 0.0
+        for col in expected_columns:
+            assert (df_result[col] > 0.0).all() == True
```

Comment:
Wondering if '==' could be replaced with 'is' for better clarity.

Reply:
To me both are clear, and it doesn't matter as long as we're consistent :)

Related: Could we automatically format python code just like `prettier` does for JS/TS?

Reply:
Either way for me. Happy to change to 'is' if that is the preferred way. Let me know which way!

Reply:
@Greenheart, there are numerous code formatters and linters for Python. This article presents a nice smorgasbord to meditate over : https://inventwithpython.com/blog/2022/11/19/python-linter-comparison-2022-pylint-vs-pyflakes-vs-flake8-vs-autopep8-vs-bandit-vs-prospector-vs-pylama-vs-pyroma-vs-black-vs-mypy-vs-radon-vs-mccabe/

Reply:
'is' is indeed preferred :)